.buildkite/lib/generate.jl

```diff
@@ -7,6 +7,13 @@ function generate(platform::Platform)
     echo "--- Print kernel information"
     uname -a
 
+    echo "--- Print environment variables"
```

Comment:
Would it be useful to include this in general? Then a separate PR would be reasonable.